### DISCUSSION TITLE
Send token to email

### DISCUSSION THREAD

**Discussion by TianaNanta** [fastapi-users/fastapi-users#1340] [Q&A]
Hello, I want some help on implementing a button to an email that executes the verify route (the POST method) with the token as data to be sent

I have very little knowledge about html js css and jinja2😓

**Comment by hgalytoby** [comment]
> Hello, I want some help on implementing a button to an email that executes the verify route (the POST method) with the token as data to be sent
> 
> I have very little knowledge about html js css and jinja2😓

chatgpt can help you.

https://chat.openai.com/

**Comment by TianaNanta** [comment]
![Screenshot from 2024-01-15 18-50-16](https://github.com/fastapi-users/fastapi-users/assets/75852322/7f1899dc-4b99-49d8-acbe-2b5d1e6a3ca9)
I have done this but it does not appear to be working

**Reply by hgalytoby** [reply]
Do you check the response message and the log message on the backend?
